iree/turbine/kernel/wave/minimize_global_loads.py

```diff
@@ -89,6 +89,14 @@ def identify_optimizable_loads(
         custom = get_custom(read_node)
         if custom.memory in processed_memories:
             continue
+
+        # Bail out if we have dynamic values dependensies with elements_per_thread != 1
+        if any(
+            (getattr(get_custom(v), "elements_per_thread", None) != 1)
+            for v in custom.mapping_dynamic_vals
+        ):
```

Comment:
Actually, I think we should skip this pass if we have dynamic vals regardless of the elements per thread because that violates the assumptions of the pass.

Reply:
https://github.com/iree-org/iree-turbine/pull/383